src/js/components/stateLanding/table/StateLinkCell.jsx

```diff
@@ -9,15 +9,29 @@ import reactStringReplace from 'react-string-replace';
 
 const propTypes = {
     name: PropTypes.string,
+    code: PropTypes.string,
     fips: PropTypes.string,
-    searchString: PropTypes.string
+    searchString: PropTypes.string,
+    isAbbrev: PropTypes.array
 };
 
 export default class StateLinkCell extends React.Component {
     render() {
         let name = this.props.name;
+        const abbreviation = this.props.isAbbrev[0];
+        const stateWithAbbreviation = `${name}, ${this.props.code}`;
```

Comment:
I would do this formatting in the model (`models/v2/state/BaseStateLandingItem.js`) so that `this.props.name` already includes the abbreviation. Also please update to use parens instead of a comma.